frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -30,11 +30,13 @@ export enum ClusterUpdateStatus {
 
 export const clusterVersionReference = referenceForModel(ClusterVersionModel);
 
-const getAvailableClusterUpdates = (cv: ClusterVersionKind): Release[] => {
-  return cv?.status?.availableUpdates || [];
+const getAvailableClusterUpdates = (cv: ClusterVersionKind): VersionUpdate[] => {
```

Comment:
Nit:  this could be simpler.

```suggestion
const getAvailableClusterUpdates = (cv: ClusterVersionKind): VersionUpdate[] => cv?.status?.availableUpdates?.map((update) => _.pick(update, ['version', 'image'])) || [];
```